test/e2e/content/case.excluded-domain.test.js

```diff
@@ -1,10 +1,45 @@
-/*globals describe, it */
+/*eslint no-unused-expressions: "off" */
+/*globals describe, before, after, it, expect, browser, unsupportedURL, popupURL, selectors */
 
 /* Ideally, we should test accessing a page in an excluded domain, such as facebook.com. However, that requires actually loading the extension, which I have not managed to do successfully. These pending tests are here as a placeholder to remind us this use case is not tested. */
 
 describe("When the page is in an excluded domain, it", function() {
+    let page;
+    let popup;
+    let originalContent;
+
+    before(async function() {
+        this.timeout(10000); // Loading Facebook can take more than 8 seconds.
+        page = await browser.newPage();
+        await page.goto(unsupportedURL);
+
+        popup = await browser.newPage();
+        await popup.goto(popupURL);
+
+        originalContent = await page.$eval("body", e => e.innerText);
+    });
+
+    after(async function() {
+        await page.close();
+        await popup.close();
+    });
+
     it("should load the Chrome extension");
-    it("should not change the text");
-    it("should explain in the header");
-    it("should not have a 'Show changes/highlights' button");
```

Comment:
Thank you so much for taking care of these tests I proposed before but could not implement.